pkg/virt-api/api.go

```diff
@@ -905,7 +905,7 @@ func (app *virtAPIApp) startTLS() error {
 		// response is given. That status request won't send a peer cert regardless
 		// if the TLS handshake requests it. As a result, the TLS handshake fails
 		// and our aggregated endpoint never becomes available.
-		ClientAuth: tls.RequestClientCert,
+		ClientAuth: tls.VerifyClientCertIfGiven,
```

Comment:
Should we use `RequireAndVerifyClientCert`?

Reply:
According to this comment, no:  https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-api/api.go#L902-L907

Reply:
For privileged urls, here is where we check that the request contains a cert: https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-api/rest/authorizer.go#L209